client/src/components/DateInput.js

```diff
@@ -32,18 +32,17 @@ const DateInput = ({
         <input
           type="checkbox"
           checked={autoCompleteCheckbox}
```

Comment:
I think that the `autoCompleteCheckbox` variable is superfluous here - it's pretty much always going to be the same as `showAutocomplete`. While there are some tradeoffs (we'd have to pass in all of the `showAutocomplete` variables through App.js which is what we've been trying to minimize) I think this is probably the best practice as it creates a single source of truth.